### ISSUE TITLE
Time-of-day switching twitchiness

### ISSUE THREAD

**Issue by clayote** [elseheartbreak/elseheartbreak.github.io#3]
I'm observing this behavior in the Linux version of the game.

When the game time is pretty close to a transition between one part of the day and the next, say midday to afternoon, the lighting conditions will sometimes flicker rapidly between the current part of the day and the next. It depends on the camera angle. I can make it go away (at least for the time being) by rotating the camera.

I originally thought this was an aesthetic choice, like the computer ministry was doing a poor job of maintaining the sun's code, but the fact that it depends on the camera angle suggests otherwise.

**Comment by eriksvedang** [comment]
Yeah, seems like the Linux build has some weirdness when it comes to lighting. Some people have experienced less trouble when switching to a lower resolution, does that help for you also?

**Comment by clayote** [comment]
That does make the flickering go away, yeah. An acceptable workaround, since the game has a low-rez sort of style anyhow.

**Comment by eriksvedang** [comment]
Yeah, it's not completely satisfactory but I haven't come up with any other good ideas for how to solve it, unfortunately :/
